meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend

```diff
@@ -2,3 +2,27 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 EXTRA_OENPM:witherspoon-tacoma = "-- --mode ibm"
 EXTRA_OENPM:p10bmc = "-- --mode ibm"
+
+inherit obmc-phosphor-systemd
+
+SRC_URI:append:p10bmc = " \
+	file://changeLogo.sh \
+	file://change-logo.service \
+	file://inspur-login-logo.svg \
+	file://inspur-logo-header.svg \
+    file://blankLogo.svg \
```

Comment:
Nit: the indentation here seems to be off.

Reply:
Done